api/resources_portal/models/notification.py

```diff
@@ -0,0 +1,92 @@
+import logging
+
+from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from computedfields.models import ComputedFieldsModel, computed
+
+from resources_portal.models.material import Material
+from resources_portal.models.organization import Organization
+from resources_portal.models.user import User
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+logger.addHandler(logging.StreamHandler())
+
+
+class Notification(ComputedFieldsModel):
+    class Meta:
+        db_table = "notifications"
+        get_latest_by = "created_at"
+
+    NOTIFICATION_TYPES = (
+        ("ORG_REQUEST_CREATED", "ORG_REQUEST_CREATED"),
+        ("ORG_INVITE_CREATED", "ORG_INVITE_CREATED"),
+        ("ORG_INVITE_ACCEPTED", "ORG_INVITE_ACCEPTED"),
+        ("ORG_REQUEST_ACCEPTED", "ORG_REQUEST_ACCEPTED"),
+        ("ORG_INVITE_REJECTED", "ORG_INVITE_REJECTED"),
+        ("ORG_REQUEST_REJECTED", "ORG_REQUEST_REJECTED"),
+        ("ORG_INVITE_INVALID", "ORG_INVITE_INVALID"),
+        ("ORG_REQUEST_INVALID", "ORG_REQUEST_INVALID"),
+        ("MTA_UPLOADED", "MTA_UPLOADED"),
+        ("APPROVE_REQUESTS_PERM_GRANTED", "APPROVE_REQUESTS_PERM_GRANTED"),
+        ("TRANSFER_REQUESTED", "TRANSFER_REQUESTED"),
+    )
+
+    objects = models.Manager()
+
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    notification_type = models.CharField(max_length=32, choices=NOTIFICATION_TYPES)
+    notified_user = models.ForeignKey(
```

Comment:
Can we also record what email we sent the notification to? This way if a user is complaining about their notifications not coming through and then changes their email we'll have a trail.

Reply:
Good point.

Reply:
So I meant can we get an `email_address` field that gets set in `send_email_notification` with whatever email address was notified?